war/package.json

```diff
@@ -39,14 +39,16 @@
     "less-loader": "11.1.0",
     "mini-css-extract-plugin": "2.7.5",
     "postcss": "8.4.21",
-    "postcss-less": "6.0.0",
     "postcss-loader": "7.2.4",
+    "postcss-scss": "^4.0.6",
     "prettier": "2.8.7",
+    "sass": "^1.62.0",
+    "sass-loader": "^13.2.2",
```

Comment:
Can we remove the carat to be consistent with the other version numbers in this file?